Sources/ApolloSQLite/SQLiteNormalizedCache.swift

```diff
@@ -19,28 +19,29 @@ public final class SQLiteNormalizedCache {
   private let record = Expression<String>("record")
   private let shouldVacuumOnClear: Bool
 
-  /// Designated initializer
-  ///
+  /// Creates a store that uses the provided SQLite file connection.
   /// - Parameters:
-  ///   - fileURL: The file URL to use for your database.
-  ///   - shouldVacuumOnClear: If the database should also be `VACCUM`ed on clear to remove all traces of info. Defaults to `false` since this involves a performance hit, but this should be used if you are storing any Personally Identifiable Information in the cache.
-  /// - Throws: Any errors attempting to open or create the database.
-  public init(fileURL: URL, shouldVacuumOnClear: Bool = false) throws {
+  ///   - db: The database connection to use.
+  ///   - shouldVacuumOnClear: If the database file should compact "VACUUM" its storage when clearing records. This can be a potentially long operation.
+  ///   Default is `false`. This SHOULD be set to `true` if you are storing any Personally Identifiable Information in the cache.
+  /// - Throws: Any errors attempting to access the database.
+  public init(db: Connection, compactFileOnClear shouldVacuumOnClear: Bool = false) throws {
     self.shouldVacuumOnClear = shouldVacuumOnClear
-    self.db = try Connection(.uri(fileURL.absoluteString), readonly: false)
+    self.db = db
     try self.createTableIfNeeded()
   }
 
-  ///
-  /// Initializer that takes the Connection to use
+  /// Creates a store that will establish its own connection to the SQLite file at the provided url.
   /// - Parameters:
-  ///   - db: The database Connection to use
-  ///   - shouldVacuumOnClear: If the database should also be `VACCUM`ed on clear to remove all traces of info. Defaults to `false` since this involves a performance hit, but this should be used if you are storing any Personally Identifiable Information in the cache.
-  /// - Throws: Any errors attempting to access the database
-  public init(db: Connection, shouldVacuumOnClear: Bool = false) throws {
-    self.shouldVacuumOnClear = shouldVacuumOnClear
-    self.db = db
-    try self.createTableIfNeeded()
+  ///   - fileURL: The file URL to use for your database.
+  ///   - shouldVacuumOnClear: If the database file should compact "VACUUM" its storage when clearing records. This can be a potentially long operation.
```

Comment:
the name of this parameter should be updated to reflect its external value

Reply:
Is that the way you prefer?

In the SSS ecosystem (and if you look at a lot of Apple code as well) they use the parameter label rather than the argument label and the Xcode Quick Help properly labels the documentation with the argument label.

I can make this change, just wanted to double check 😄

Reply:
oh! I've been updating it manually like a sucker 🙃

Reply:
I'm a liar. I just tested and I was mistaken - Quick Help does _not_ use the argument label and instead uses the parameter label in documentation.

_However_, usually the parameter label tends to be more descriptive in a documentation context on how the property will be used, while the argument label would not.

For example, from [RediStack](https://gitlab.com/Mordil/RediStack/-/blob/833fc338816746d5b532a66a3a3129649ef64f9e/Sources/RediStack/Commands/BasicCommands.swift#L85):
```swift
public func authorize(with password: String) -> EventLoopFuture<Void>

// call site
client.authorize(with: myPassword)

// documentation
password
The password to authenticate with.
```

I guess it's definitely up to personal preference, and I'm happy to update these to match Apollo's preference

Reply:
Ahhh, yeah I suppose that's fair. Yeah just leave it. IGNORE ME.